buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -6,19 +6,19 @@ object Versions {
    * Source
    */
   val javaTarget = JavaVersion.VERSION_1_8
-  const val kotlin = "1.9.10"
+  const val kotlin = "2.0.0"
```

Comment:
Kotlin 2.0.0 is very new, sometimes it's good to wait a few minor/patch versions before upgrading. But if we do want to upgrade, [it looks like](https://kotlinlang.org/docs/whatsnew20.html#current-k2-compiler-limitations) we should also update to a more recent Gradle (8.3+) as well

Reply:
Oops looks like I was looking at an old branch, we're already using gradle 8.4 🤦

Reply:
Yeah, unless there's a reason to move to 2.0. Probably stay on 1.9.x.

Upgrading to 1.9.20 seems fine.

Reply:
I changed it to 1.9.20